### PR TITLE
Blake griswell introduction meme

### PR DESCRIPTION
**Pull request by sflowers51**

### CODE REVIEW

File: _posts/2023-01-24-BlakeGriswell-introduction-meme.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (Soyokaze-42) [inline _posts/2023-01-24-BlakeGriswell-introduction-meme.md:1]: You need to update this file with the front matter and the description of your meme. Please keep it relevant to the introduction chapter of our book. The reading did not discuss shells or real-world coding.

Comment (BlakeGriswell) [reply _posts/2023-01-24-BlakeGriswell-introduction-meme.md:1]: Ok,I think this one is better.

File: _posts/2023-01-24-BlakeGriswell-introduction-meme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+title:  Left Exit 12 Off Ramp
+author: Blake Griswell
```
Comment (Soyokaze-42) [inline _posts/2023-01-24-BlakeGriswell-introduction-meme.md:3]: The author here needs to be your short_name: Griz.

Comment (BlakeGriswell) [reply _posts/2023-01-24-BlakeGriswell-introduction-meme.md:3]: Ok, Thanks I guess I did not catch that before.

**Review by Soyokaze-42 — APPROVED** [review]
Looks good!

**Pull request merged**